charts/opentelemetry-demo/values.yaml

```diff
@@ -93,13 +93,15 @@ components:
     # If a different url is desired, set DATABASE_URL here.
     # - name: DATABASE_URL
     #   value:
-    - name: GRPC_PORT
+    - name: FEATURE_FLAG_GRPC_SERVICE_PORT
+      value: '8080'
+    - name: FEATURE_FLAG_SERVICE_PORT
       value: '50053'
-    - name: PORT
-      value: '50052'
+    - name: SECRET_KEY_BASE
+      value: 'Vive++6s/oRCd7xI3zSvyUGH1AJQS0BE3xjNrEOJEVmzOg5qPBlTYE+5kgz2cfBE'
```

Comment:
Is this required?

Reply:
It can probably be removed as a requirement from the underlying app.

Reply:
Secrets aren't a good thing to include in a chart by default typically.  If a secret needs set it is better to use helm's `--set` flag when running `helm install` to set a secret. Can we remove it from this PR?

Reply:
[It looks like there is a value checked into source control](https://github.com/open-telemetry/opentelemetry-demo/blob/3ccd221fafefdb3c8b3fdcc302ad03ed5dca1f6f/src/featureflagservice/Dockerfile#L88) in the demo so maybe this env var is ok, but I'd rather see the values match.  Can you reach out in the Demo Slack channel and confirm this value should be committed and that its safe to include in the helm charts.

Reply:
Will do.

Reply:
@joshleecreates Looks like the value isn't needed in the helm charts env var bc it is built into the docker container the chart relies on. I think we can safely remove this.

Reply:
Ah makes sense, I was testing directly without Docker. This is not present on the current version of the PR, should be gtg